### PR TITLE
[DOCS] Add ml-cpp PRs to 7.7 release notes

### PR DESCRIPTION
**Pull request by lcawl**
This PR adds details from https://github.com/elastic/ml-cpp/blob/7.7/docs/CHANGELOG.asciidoc to the 7.7 Elasticsearch Release Notes (https://www.elastic.co/guide/en/elasticsearch/reference/7.7/release-notes-7.7.0.html).
It also removes machine learning PRs from the list that were already shipped in 7.6.x releases.

### CODE REVIEW

File: docs/reference/release-notes/7.7.asciidoc [hunk 1]
```diff
@@ -283,8 +286,32 @@ Machine Learning::
 * Add tags url param to GET {pull}51330[#51330]
 * Add parsers for inference configuration classes {pull}51300[#51300]
 * Make datafeeds work with nanosecond time fields {pull}51180[#51180] (issue: {issue}49889[#49889])
-* Add audit warning for 1000 categories found early in job {pull}51146[#51146] (issue: {issue}50749[#50749])
 * Adds support for a global calendars {pull}50372[#50372] (issue: {issue}45013[#45013])
+* Improve computational performance of the feature importance computation
```
Comment (droberts195) [inline docs/reference/release-notes/7.7.asciidoc:290]:
```suggestion
* Speed up computation of feature importance
```

**Review by droberts195 — APPROVED** [review]
LGTM

**Comment by lcawl** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by lcawl** [comment]
@elasticmachine update branch

**Pull request merged**